Dockerfile

```diff
@@ -26,6 +26,10 @@ RUN git clone --depth=1 https://github.com/containernetworking/plugins.git $GOPA
     git checkout tags/${TAG} -b ${TAG} &&\
     go mod download
 
+RUN git clone --depth=1 https://github.com/k8snetworkplumbingwg/bond-cni.git $GOPATH/src/github.com/k8snetworkplumbingwg/bond-cni && \
```

Comment:
Even if they are not releasing, can we at least pin to a commit? Otherwise, the builds will not be reproducible